release-kong.sh

```diff
@@ -64,9 +64,7 @@ function push_docker_images() {
     "$DOCKER_REPOSITORY:$ARCHITECTURE-$KONG_VERSION"
 
   echo "FROM $DOCKER_REPOSITORY:$ARCHITECTURE-$KONG_VERSION" | docker build \
-    --label org.opencontainers.image.version="$KONG_VERSION" \
-    --label org.opencontainers.image.created="$DOCKER_LABEL_CREATED" \
-    --label org.opencontainers.image.revision="$DOCKER_LABEL_REVISION" \
+    ${DOCKER_LABELS} \
```

Comment:
in theory shouldn't be necessary. This image is built by `build-test-container` and this step used to add labels to that container.

In hindsight we always should have labelled the container when building and not bothered with adding labels in a subsequent step. I'm not sure how safe it is to remove now :(